frameworks/digital-outcomes-and-specialists-2/manifests/edit_brief_response.yml

```diff
@@ -1,11 +1,10 @@
 -
-  name: Apply for this opportunity
+  name: Apply for an opportunity
```

Comment:
Why the change to 'an'? On the buyer frontend `opportunities/brief_id` page we use 'this' still?